### PR TITLE
Update docs: PVs and storage classes

### PR DESCRIPTION
**Pull request by serenashe**
Signed-off-by: adakisme <serenashe@yunify.com>
I updated pics and some descriptions. Please take a look, thanks.
/assign @Patrick-LuoYu @Felixnoo

### CODE REVIEW

File: content/en/docs/cluster-administration/persistent-volume-and-storage-class.md [hunk 1]
```diff
@@ -32,29 +32,26 @@ You need an account granted a role including the authorization of **Cluster Mana
...
 
     ![storage-class](/images/docs/cluster-administration/persistent-volume-and-storage-class/storage-class.jpg)
 
-4. To create a storage class, click **Create** and enter the basic information in the pop-up window. When you finish, click **Next**.
+4. To create a storage class, click **Create** and enter the basic information in the displayed dialog box. When you finish, click **Next**.
 
-    ![create-storage-class-basic-info](/images/docs/cluster-administration/persistent-volume-and-storage-class/create-storage-class-basic-info.png)
-
-5. In KubeSphere, you can create storage classes for `QingCloud-CSI`, `Glusterfs` and `Ceph RBD` directly. Alternatively, you can also create customized storage classes for other storage systems based on your needs. Select a type and click **Next**.
+5. In KubeSphere, you can create storage classes for `QingCloud-CSI`, `Glusterfs` and `Ceph RBD`. Alternatively, you can also create customized storage classes for other storage systems based on your needs. Select a type and click **Next**.
```
Comment (Felixnoo) [inline content/en/docs/cluster-administration/persistent-volume-and-storage-class.md:45]: Glusterfs and -> Glusterfs, and

File: content/en/docs/cluster-administration/persistent-volume-and-storage-class.md [hunk 2]
```diff
@@ -77,18 +74,18 @@ QingCloud CSI is a CSI plugin on Kubernetes for the storage service of QingCloud
 
 #### Settings
 
-![storage-volume-qingcloud](/images/docs/cluster-administration/persistent-volume-and-storage-class/storage-volume-qingcloud.jpg)
+![storage-volume-qingcloud](/images/docs/cluster-administration/persistent-volume-and-storage-class/storage-volume-qingcloud.png)
 
 | Property | Description |
 | :---- | :---- |
-| type     | On the QingCloud platform, 0 represents high performance volumes. 2 represents high capacity volumes. 3 represents super high performance volumes. 5 represents Enterprise Server SAN. 6 represents NeonSan HDD. 100 represents standard volumes. 200 represents enterprise SSD. |
+| type     | On the QingCloud platform, 0 represents high performance volumes. 2 represents high capacity volumes. 3 represents super high performance volumes. 5 represents Enterprise Server SAN. 100 represents standard volumes. 200 represents enterprise SSD. |
```
Comment (Felixnoo) [inline content/en/docs/cluster-administration/persistent-volume-and-storage-class.md:81]: Please confirm the deletion here.

Comment (serenashe) [reply content/en/docs/cluster-administration/persistent-volume-and-storage-class.md:81]: Confirmed. The original sentence should be removed.

File: content/en/docs/cluster-administration/persistent-volume-and-storage-class.md (hunk 1 above)
Comment (serenashe) [reply content/en/docs/cluster-administration/persistent-volume-and-storage-class.md]: Changed.

**Pull request merged**